### PR TITLE
Add machineClasses to default kustomization config list

### PR DESCRIPTION
**Pull request by enxebre**
**What this PR does / why we need it**:
Add machineClasses to default kustomization config list

### CODE REVIEW

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ resources:
 - ../crds/cluster_v1alpha1_machine.yaml
 - ../crds/cluster_v1alpha1_machinedeployment.yaml
 - ../crds/cluster_v1alpha1_machineset.yaml
+- ../crds/cluster_v1alpha1_machineclass.yaml
```
Comment (roberthbailey) [inline config/default/kustomization.yaml]: These look alphabetically sorted, in which case this should go above machinedeployment.yaml.

**Comment by roberthbailey** [comment]
Ping @enxebre - it'd be nice to get this in before we cut a new build.

**Comment by enxebre** [comment]
Sorry @roberthbailey I'm travelling these days with no internet access most of the time. Updated.

**Pull request merged**